src/OrchardCore/OrchardCore.Abstractions/Modules/Extensions/StringExtensions.cs

```diff
@@ -7,11 +7,6 @@ namespace OrchardCore.Modules
 {
     public static class StringExtensions
     {
-        public static string ToHexString(this byte[] bytes)
-        {
-            return BitConverter.ToString(bytes).Replace("-", "");
-        }
-
```

Comment:
Why remove this?

Reply:
Not used anywhere, and there is a built-in one in the BCL.